compiler/rustc_const_eval/src/interpret/memory.rs

```diff
@@ -877,10 +877,10 @@ impl<'tcx, M: Machine<'tcx>> InterpCx<'tcx, M> {
         if let Some(fn_val) = self.get_fn_alloc(id) {
             let align = match fn_val {
                 FnVal::Instance(instance) => {
-                    // Function alignment can be set globally with the `-Zmin-function-alignment=<n>` flag;
-                    // the alignment from a `#[repr(align(<n>))]` is used if it specifies a higher alignment.
+                    // Function alignment can be set globally with the `-Cmin-function-alignment=<n>` flag;
+                    // the alignment from a `#[align(<n>)]` is used if it specifies a higher alignment.
                     let fn_align = self.tcx.codegen_fn_attrs(instance.def_id()).alignment;
-                    let global_align = self.tcx.sess.opts.unstable_opts.min_function_alignment;
+                    let global_align = self.tcx.sess.opts.cg.min_function_alignment;
```

Comment:
(orthogonal to this PR)
it is not great that the logic for merging the per-fn alignment and the global alignment needs to be repeated in each backend.

Maybe `codegen_fn_attrs` should just take `min_function_alignment` into account?